docs/en/administration/management/BE_blacklist.md

```diff
@@ -2,24 +2,26 @@
 displayed_sidebar: docs
 ---
 
-# Manage BE Blacklist
+# Manage BE and CN Blacklist
 
-This topic describes how to manage the BE Blacklist.
+This topic describes how to manage the BE and CN Blacklist.
```

Comment:
Let's not use "this topic" anymore. The title of the doc is enough in this case.
```suggestion
```